app/javascript/listings/listings.jsx

```diff
@@ -360,21 +395,30 @@ export class Listings extends Component {
           id="classified-listings-modal-background"
         />
       );
-      if (
-        openedListing.contact_via_connect &&
-        openedListing.user_id !== currentUserId
```

Comment:
why was this removed and moved down?

Reply:
I tried to avoid code duplicates in that part.
The original one repeat the whole <form> when the only difference are the two texts.

The condition contains `openedListing.contact_via_connect` in either case so I move it up, and also for me to use the `? ... : ...` syntax.

Reply:
I was trying to remove duplicate. The whle <form> is repeated even though the only differences are the two texts

Both case of the condition contains `openedListing.contact_via_connect` so I put that to the top, and that also allows me to use ` ? ... : ...` syntax.